index.html

```diff
@@ -1,11 +1,176 @@
 <!DOCTYPE html>
 <html lang="en" dir="ltr">
   <head>
-    <meta charset="utf-8">
-    <link rel="stylesheet" href="/style.css">
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="./reset.css" />
+    <link rel="stylesheet" href="./style.css" />
+    <!-- <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
```

Comment:
**[REQUIRED]** Remove the commented code

Reply:
Thanks Wathigo, We're gonna make those changes you suggest!

Reply:
Changes done!